AssemblerLib/VectorMatrixAssembler.h

```diff
@@ -136,16 +136,20 @@ class VectorMatrixAssembler<GlobalMatrix, GlobalVector, LocalAssembler,
     /// \attention The index \c id is not necessarily the mesh item's id.
     void preTimestep(std::size_t const id,
                      LocalAssembler& local_assembler,
-                     GlobalVector const& x) const
+                     GlobalVector const& x,
+                     double const t,
```

Comment:
Agree with Christoph, as long as we don't need the whole TimeStep object (e.g. for the multi-step methods storing multiple dt's) the two doubles are good.